tests/runtests.py

```diff
@@ -396,8 +413,9 @@ def paired_tests(paired_test, options, test_labels, parallel):
              'LiveServerTestCase) is expected to run from. The default value '
              'is localhost:8081-8179.')
     parser.add_argument(
-        '--selenium', action='store_true', dest='selenium', default=False,
-        help='Run only the Selenium tests (equivalent to "--tag selenium").')
+        '--selenium', dest='selenium', action=ActionSelenium,
+        help='A comma separated list of browsers to run the Selenium test against. '
+             'You can run only the Selenium tests by adding "--tag selenium".')
```

Comment:
@timgraham I changed this argument meaning a bit to allow running the full test suite (selenium tests included) like it was possible before this change. Could you adjust the selenium PR builder `EXTRA_RUNTESTS_ARGS` to pass the `--tag selenium` to prevent the full test suite from running. Thanks!